definitions/ext-sap_idoc/definition.yml

```diff
@@ -0,0 +1,27 @@
+domain: EXT
+type: SAP_IDOC
+goldenTags:
+    - instrumentation.provider
+    - SAP.FUNCTIONAL_AREA
+    
+synthesis:
+  rules:
+    # telemetry from New Relic SAP intergraion
+    # SAP_EID in format of SYS_ID/MESSAGE_TYPE
+  - identifier: SAP_EID
+    name: MESSAGE_TYPE_DESCRIPTION
+    encodeIdentifierInGUID: true
+    conditions:
+      - attribute: SAP_ETYPE
+        value: IDOC
+
+    tags:
+      SAP.FUNCTIONAL_AREA:
```

Comment:
Is there any reason to make tags all upper case? is this a sap standard? or is something our integration does?

Reply:
It's SAP conversion to make parameter names upper case. In this case, however, the tag is generated by the integration and controlled by us. We updated the tag to be low case.